### ISSUE TITLE
EMBY: This website is not supported

### ISSUE THREAD

**Issue by janaxhell** [web-scrobbler/web-scrobbler#4951]
**Describe the bug**
Despite having set the URL pattern of my tld (I exposte Emby via cloudflare domain), Webscrobbler says it's not supported

**How to reproduce**
1. Go to Connectors
2. Choose Emby/Jellyfin
3. set URL pattern
4. Start any music video from Emby tld
5. click the WS extension

**Expected behavior**
Scrobbling the tracks

**Screenshots**
![vivaldi_bRy39J4Hxe](https://github.com/user-attachments/assets/e9715ce7-3606-4add-9feb-59db5ff067a3)

![vivaldi_X66djfVat9](https://github.com/user-attachments/assets/5876ea21-91dc-4a29-93f0-c271360d0bb9)

**Debug logs**
https://pastebin.com/DgvMuYUd

**Environment (please complete the following information)**
-   OS: Windows 10
-   Browser: Vivaldi
-   Extension version: v3.10.0

**Additional context**
I'm trying this from my PC, from my browser, not from an Android app on TV. 
Youtube on the same browser works fine.

**Comment by disgustipated** [comment]
I ran into this on mine, I suspect something was cached in the browser when I put the url in because once i went to try debugging the extension by enabling console debugging, i refreshed the page with console dev tools open and it began working.

**Comment by rob9315** [comment]
Yes, settings changes only apply after reloading the page. the content-script only checks the url once when it is initially loaded, please check if [the wiki article on url-patterns](https://github.com/web-scrobbler/web-scrobbler/wiki/Custom-URL-patterns) helps you and fixes your issue. If not, feel free to reopen this issue
